zvariant_derive/src/lib.rs

```diff
@@ -424,6 +424,48 @@ pub fn deserialize_dict_macro_derive(input: TokenStream) -> TokenStream {
 /// assert_eq!(e, StrEnum::Variant2);
 /// ```
 ///
+///# Auto renaming fields
+///
+/// The macro supports specifying a Serde-like `#[zvariant(rename_all = "case")]` attribute on
+/// structures. The attribute allows to rename all the fields from snake case to another case
```

Comment:
Really nice job with documentation! So I hate to ask for any more changes but I'm a perfectionist. :see_no_evil: It looks like we don't already document the renaming of fields so probably best to generalize the header and text here to introduce to both kinds of renaming?

Reply:
I just copied this from the `SerializeDict` docs. I'll definitely refine this to make sure both attributes are well explained.